spec/controllers/spree/admin/general_settings_controller_decorator_spec.rb

```diff
@@ -26,17 +22,17 @@
   describe "GET 'edit'" do
 
     it "assigns @preferences_loyalty_points" do
```

Comment:
@jatin-baweja For the following specs, can we move them to context block and add a before block which makes the edit request?